### PR TITLE
Animated phoenix

### PR DESCRIPTION
**Pull request by maretamara**
Wprawiam gwiazdy w ruch pod wpływem myszki 

* [x] @beatadelura 
* [x] @kruszczynski 
* [ ] @MagdaMalinowska 
* [ ] @AdamMankowski 
* [ ] @mikolajgrzyb 
* [ ] @rpietraszko 
* [ ] @pjar

### CODE REVIEW

File: .bash_profile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+bind "set show-all-if-ambiguous on"
+if [ -f $(brew --prefix)/etc/bash_completion ]; then
+    . $(brew --prefix)/etc/bash_completion
+fi
```
Comment (beatadelura) [inline .bash_profile]: This file should be removed.

File: bash [hunk 2]
```diff
@@ -0,0 +1,3 @@
+f [ -f ~/.git-completion.bash ]; then
+  . ~/.git-completion.bash
+fi
```
Comment (beatadelura) [inline bash]: This file should be removed.

File: source/javascripts/phoenix-animate.js.coffee [hunk 3]
```diff
@@ -0,0 +1,15 @@
+$ ->
+	$(".job-bcg").interactive_bg ->
+		console.log('dssaddsa')
```
Comment (beatadelura) [inline source/javascripts/phoenix-animate.js.coffee]: Please remove `console.log` :)

File: source/join-our-crew/elixir.html.slim [hunk 4]
```diff
@@ -1,73 +1,76 @@
...
-            involvement (8000+ PLN net)
-        li Private medical insurance
-        li Sports benefits system
-        li Employment form of your choice
-        li Flexible hours, with occasional possibility of remote work
-        li Interesting projects for international clients
-        li Experienced, diverse team
-        li Thoroughly modern stack, with room for experimentation
-        li Office in Warsaw next to the Imielin metro station
+  .div(style="height: 2466px;")
```
Comment (beatadelura) [inline source/join-our-crew/elixir.html.slim]: Please rename this class, e.g. `job-offer-description`. Why have inline styles been used?

File: source/stylesheets/layouts/_l-landing-elixir.scss [hunk 5]
```diff
@@ -1,13 +1,28 @@
 .l-elixir {
   background: $mirage-blue;
-  background-image: url('../svg/elixir/bcg_elixir.svg');
+  /*background-image: url('../svg/elixir/bcg_elixir.svg');*/
```
Comment (beatadelura) [inline source/stylesheets/layouts/_l-landing-elixir.scss]: Maretka, please remove 3, 150 and 213 line ;)

File: source/stylesheets/layouts/_l-landing-elixir.scss [hunk 6]
```diff
@@ -1,13 +1,28 @@
...
   background-size: 100%;
   color: $white;
   font-family: $baskerville;
   font-size: 22px;
 
+  .ibg-bg {
+    position: absolute;
+  }
+
+  .job-ofert {
```
Comment (beatadelura) [inline source/stylesheets/layouts/_l-landing-elixir.scss]: This class should name `job-offer`, I supposed.

File: source/stylesheets/layouts/_l-landing-elixir.scss [hunk 7]
```diff
@@ -16,15 +31,8 @@
...
-  .apply-link {
-    @include font(34px, 0px);
-    text-transform: lowercase;
-    text-align: center;
-    display: block;
-  }
-
   .arrow {
-    height: 110px;
+    height: 11-px;
```
Comment (beatadelura) [inline source/stylesheets/layouts/_l-landing-elixir.scss]: It won't work :)

File: source/stylesheets/layouts/_l-landing-elixir.scss [hunk 8]
```diff
@@ -113,7 +128,7 @@
     vertical-align: middle; 
     background-color: #181b28;
     padding: 7px;
-    margin: 15px 0px 40px 0px;
+    margin: 15px 0px 15px 0px;
```
Comment (beatadelura) [inline source/stylesheets/layouts/_l-landing-elixir.scss]: Please use shorthand :)

**Comment by beatadelura** [comment]
We have to fix also `js.coffee` file :)

File: source/javascripts/phoenix-animate.js.coffee [hunk 9]
```diff
@@ -0,0 +1,11 @@
+$ ->
+	$(".job-bcg").interactive_bg ->
```
Comment (MagdaMalinowska) [inline source/javascripts/phoenix-animate.js.coffee]: pls correct indentation:)

Comment (maretamara) [inline source/javascripts/phoenix-animate.js.coffee]: poprawione :)

File: source/join-our-crew/elixir.html.slim (hunk 4 above)
Comment (maretamara) [inline source/join-our-crew/elixir.html.slim]: poprawione

File: source/stylesheets/layouts/_l-landing-elixir.scss (hunk 5 above)
Comment (maretamara) [inline source/stylesheets/layouts/_l-landing-elixir.scss]: zostało to zmienione, zostaje na mobilkach i tabletach a w desktop jest usunięte

File: source/stylesheets/layouts/_l-landing-elixir.scss (hunk 6 above)
Comment (maretamara) [inline source/stylesheets/layouts/_l-landing-elixir.scss]: poprawione

File: source/stylesheets/layouts/_l-landing-elixir.scss (hunk 8 above)
Comment (maretamara) [inline source/stylesheets/layouts/_l-landing-elixir.scss]: poprawione

File: source/stylesheets/layouts/_l-landing-elixir.scss (hunk 7 above)
Comment (maretamara) [inline source/stylesheets/layouts/_l-landing-elixir.scss]: wywalone

File: source/javascripts/phoenix-animate.js.coffee (hunk 3 above)
Comment (maretamara) [inline source/javascripts/phoenix-animate.js.coffee]: wywalone

**Comment by beatadelura** [comment]
bitmoji

File: source/javascripts/scroll_down_arrow.js.coffee [hunk 10]
```diff
@@ -1,8 +1,11 @@
...
+        $('[data-js=scroll-down-arrow]').fadeOut()
+      else
+        $('[data-js=scroll-down-arrow]').fadeIn()
 
-  $(window).scroll ->
-    if ($(this).scrollTop() > 0)
-      scrollDownArrow.fadeOut()
-    else
-      scrollDownArrow.fadeIn()
+    if $(window).height() > 947
```
Comment (kruszczynski) [inline source/javascripts/scroll_down_arrow.js.coffee]: This will never happen because it's inside `if height < 947`

**Pull request merged**